cypress/integration/a11y.spec.js

```diff
@@ -79,38 +72,35 @@ describe('Check accessiblity', () => {
         });
 
         context('eApp eligibility questions', () => {
-            it('Which service would you like?', () => {
-                checkA11y();
-            });
+            it('Which service would you like?', () => {});
 
             it('[Error] Which service would you like?', () => {
                 clickContinueBtn();
-                checkA11y();
+            });
+
+            it('Select radio option and check a11y', () => {
+                findByRole('radio', {name: 'e-Apostille service'}).check();
```

Comment:
why doesn't `findByLabelText` work here?

Reply:
So I read your response to this question https://github.com/UKForeignOffice/loi-application-service/pull/339#discussion_r805421555

As the role should be used to check if the element is findable, but I could be wrong.

Reply:
for form inputs/labels, they recommend findByLabelText since this is how users (screenreader/otherwise) navigate through a form 
https://testing-library.com/docs/queries/about/#priority